### PR TITLE
First release

### PR DESCRIPTION
**Pull request by marians**
First bit of code, documentation and meta stuff

### CODE REVIEW

**Comment by marians** [comment]
@zeisss Care to review?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+
+Install required python libraries:
+
+```nohighlight
+$ pip install -r requirements.txt
+```
+
+## Configuration
+
+Copy the provided `config.yaml.dist` to `config.yaml` and edit to your needs. Then edit `config.yaml` according to your needs.
```
Comment (zeisss) [inline README.md]: doppeltgemoppelt. doppeltgemoppelt.

**Comment by marians** [comment]
@zeisss Any result yet from your review?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+$ pip install -r requirements.txt
+```
+
+## Configuration
+
+Copy the provided `config.yaml.dist` to `config.yaml` and edit according to your needs.
+
+- `global.github_api_token`: Optionally set the value of this key to a valid Github API token. Without a token, you might run into rate limits when using Seismograph frequently.
+- `global.projects`: For each software project you want to track, add one key here. Name this key however you want, but make sure it's unique within this file.
+- `global.projects.<project>.github`: This key is required per project. The value must be a string composed of the user or organization name, then a forward slash, then the actual repository name. For example, the [Bootstrap](https://github.com/twbs/bootstrap) repository would get the value `twbs/bootstrap`.
```
Comment (zeisss) [inline README.md]: ~~I would put required keys at the top.~~

File: README.md [hunk 3]
```diff
@@ -1,2 +1,65 @@
...
+```nohighlight
+$ pip install -r requirements.txt
+```
+
+## Configuration
+
+Copy the provided `config.yaml.dist` to `config.yaml` and edit according to your needs.
+
+- `global.github_api_token`: Optionally set the value of this key to a valid Github API token. Without a token, you might run into rate limits when using Seismograph frequently.
+- `global.projects`: For each software project you want to track, add one key here. Name this key however you want, but make sure it's unique within this file.
```
Comment (zeisss) [inline README.md]: All your examples have the `projects` at the top of the file, not inside `global`

File: fetch.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        content = handle.read()
+        data = yaml.load(content, Loader=yaml.Loader)
+        return data
+
+
+def github_stats(client, organization, repository_name):
+    github_org = g.get_organization(organization)
+    repo = github_org.get_repo(repository_name)
+    stats = {
+        "forks_count": repo.forks_count,
```
Comment (zeisss) [inline fetch.py]: all those `_count` seems repetative. We could drop them?

**Comment by marians** [comment]
Updated according to review.

**Pull request merged**